test/report/report.test.ts

```diff
@@ -62,6 +62,42 @@ describe("report", () => {
                         },
                     ],
                     filters: [
+                        {
```

Comment:
we should move these mocks out of the test body

Reply:
Fixed in 6d956b9f1cb0f7990ec1bda2949c21280d484364. Mock is in fixture file and expected part in the snapshot.